{
    "cve": {
        "id": "CVE-2024-54514",
        "sourceIdentifier": "product-security@apple.com",
        "published": "2024-12-12T02:15:31.643",
        "lastModified": "2024-12-12T02:15:31.643",
        "vulnStatus": "Received",
        "cveTags": [],
        "descriptions": [
            {
                "lang": "en",
                "value": "The issue was addressed with improved checks. This issue is fixed in watchOS 11.2, tvOS 18.2, macOS Sequoia 15.2, iOS 18.2 and iPadOS 18.2, macOS Ventura 13.7.2, macOS Sonoma 14.7.2. An app may be able to break out of its sandbox."
            },
            {
                "lang": "es",
                "value": "El problema se solucion\u00f3 con comprobaciones mejoradas. Este problema se solucion\u00f3 en watchOS 11.2, tvOS 18.2, macOS Sequoia 15.2, iOS 18.2 y iPadOS 18.2, macOS Ventura 13.7.2, macOS Sonoma 14.7.2. Es posible que una aplicaci\u00f3n pueda salir de su zona protegida."
            }
        ],
        "metrics": {},
        "references": [
            {
                "url": "https://support.apple.com/en-us/121837",
                "source": "product-security@apple.com"
            },
            {
                "url": "https://support.apple.com/en-us/121839",
                "source": "product-security@apple.com"
            },
            {
                "url": "https://support.apple.com/en-us/121840",
                "source": "product-security@apple.com"
            },
            {
                "url": "https://support.apple.com/en-us/121842",
                "source": "product-security@apple.com"
            },
            {
                "url": "https://support.apple.com/en-us/121843",
                "source": "product-security@apple.com"
            },
            {
                "url": "https://support.apple.com/en-us/121844",
                "source": "product-security@apple.com"
            }
        ]
    }
}